ansible/roles/postgresql/tasks/main.yml

```diff
@@ -1,6 +1,26 @@
 ---
 # roles/postgresql/tasks/main.yml
 
+- name: Add PosgreSQL apt repo
```

Comment:
cool does this mean it'll auto upgrade the postgres version if we run this? not sure if it makes sense to lock to certain version or not